src/components/TileCatalog/TileCatalog.jsx

```diff
@@ -131,6 +131,7 @@ const TileCatalog = ({
                 : tile.value}
             </RadioTile>
           ))}
+          className={className}
```

Comment:
Hmm. I added that because I noticed that TileGroup was expecting it (though not using it).
I see what you mean. I think with it not being there before, it would be safe to take out.

Reply:
It looks like its there for a consumer to use the TileGroup on its own, but @tay1orjones and @davidicus may know better

Reply:
I removed it, but kept it in the TileGroup component for the consumer, unless advised otherwise.